ManifoldCAD: error on "Import Manifold" example in chromium-based browsers

I tried the new "Import Manifold" example in a few browsers:
1. Firefox: works
2. Chromium: fails
3. Google Chrome: fails
4. Brave: fails

I have not yet delved into why this is the case, but I thought to just open an issue and ping @tonious :)

The full console log after hitting "run":
```
manifoldCAD.js:1826 Running...
worker-DvMRCUPf.js:757 Bundling code took 0.17 seconds
worker-DvMRCUPf.js:757 TypeError: s.bytes is not a function
    at Ap (worker-DvMRCUPf.js:757:16154)
    at async s (worker-DvMRCUPf.js:757:17915)Caused by: TypeError: s.bytes is not a function
    at Qs (worker-DvMRCUPf.js:757:8838)
    at async ep (worker-DvMRCUPf.js:757:8587)
    at async Qd (worker-DvMRCUPf.js:757:7815)
    at async _g (worker-DvMRCUPf.js:757:8208)
    at async csg (eval at Ap (worker-DvMRCUPf.js:757:15582), <anonymous>:36:16)
    at async Ap (worker-DvMRCUPf.js:757:15685)
    at async s (worker-DvMRCUPf.js:757:17915)
e @ worker-DvMRCUPf.js:757
s @ worker-DvMRCUPf.js:757
await in s
Cp.self.onmessage @ worker-DvMRCUPf.js:757
worker-DvMRCUPf.js:757 Caused by: TypeError: s.bytes is not a function
    at Qs (worker-DvMRCUPf.js:757:8838)
    at async ep (worker-DvMRCUPf.js:757:8587)
    at async Qd (worker-DvMRCUPf.js:757:7815)
    at async _g (worker-DvMRCUPf.js:757:8208)
    at async csg (eval at Ap (worker-DvMRCUPf.js:757:15582), <anonymous>:36:16)
    at async Ap (worker-DvMRCUPf.js:757:15685)
    at async s (worker-DvMRCUPf.js:757:17915)
e @ worker-DvMRCUPf.js:757
s @ worker-DvMRCUPf.js:757
await in s
Cp.self.onmessage @ worker-DvMRCUPf.js:757
worker-DvMRCUPf.js:757 manifoldStack: TypeError: s.bytes is not a function
    at async (:35:7)
e @ worker-DvMRCUPf.js:757
s @ worker-DvMRCUPf.js:757
await in s
Cp.self.onmessage @ worker-DvMRCUPf.js:757
```

--- a/bindings/wasm/lib/export-3mf.ts
+++ b/bindings/wasm/lib/export-3mf.ts
@@ -79,8 +79,7 @@ const defaultHeader: Header = {
  * @returns A blob containing the converted model.
  */
 export async function toArrayBuffer(
-    doc: GLTFTransform.Document,
-    header: Header = {}): Promise<Uint8Array<ArrayBufferLike>> {
+    doc: GLTFTransform.Document, header: Header = {}): Promise<ArrayBuffer> {
   const to3mf = {
     meshes: [],
     components: [],
@@ -196,11 +195,11 @@ export async function toArrayBuffer(
   files[fileForContentTypes.name] = strToU8(fileForContentTypes.content);
   files[fileForRelThumbnail.name] = strToU8(fileForRelThumbnail.content);
   const zipFile = zipSync(files);
-  return zipFile;
+  return zipFile.buffer as ArrayBuffer;
 };
 
 export async function toBlob(doc: GLTFTransform.Document, header: Header = {}) {
-  const buffer: Uint8Array<ArrayBufferLike> = await toArrayBuffer(doc, header);
-  return new Blob(
-      [buffer as Uint8Array<ArrayBuffer>], {type: supportedFormat.mimetype});
+  const buffer =
+      new Uint8Array(await toArrayBuffer(doc, header) as ArrayBuffer);
+  return new Blob([buffer], {type: supportedFormat.mimetype});
 }

--- a/bindings/wasm/lib/export-model.ts
+++ b/bindings/wasm/lib/export-model.ts
@@ -44,8 +44,7 @@ interface Format {
 
 export interface Exporter {
   exportFormats: Array<Format>;
-  toArrayBuffer:
-      (doc: GLTFTransform.Document) => Promise<Uint8Array<ArrayBufferLike>>;
+  toArrayBuffer: (doc: GLTFTransform.Document) => Promise<ArrayBuffer>;
 }
 
 /**
@@ -118,7 +117,7 @@ export async function toBlob(
     doc: GLTFTransform.Document, identifier: string): Promise<Blob> {
   const format = getFormat(identifier);
   const buffer = await getExporter(identifier).toArrayBuffer(doc);
-  return new Blob([buffer as Uint8Array<ArrayBuffer>], {type: format.mimetype});
+  return new Blob([buffer], {type: format.mimetype});
 }
 
 /**
@@ -130,8 +129,7 @@ export async function toBlob(
  * @group Low Level Functions
  */
 export async function toArrayBuffer(
-    doc: GLTFTransform.Document,
-    identifier: string): Promise<Uint8Array<ArrayBufferLike>> {
+    doc: GLTFTransform.Document, identifier: string): Promise<ArrayBuffer> {
   return await getExporter(identifier).toArrayBuffer(doc);
 }
 
@@ -149,5 +147,5 @@ export async function writeFile(
 
   const exporter = getExporter(options.mimetype ?? filename);
   const buffer = await exporter.toArrayBuffer(doc);
-  return await fs.writeFile(filename, buffer);
+  return await fs.writeFile(filename, new Uint8Array(buffer));
 }

--- a/bindings/wasm/lib/gltf-io.ts
+++ b/bindings/wasm/lib/gltf-io.ts
@@ -450,10 +450,10 @@ const getIO = (): GLTFTransform.PlatformIO => {
 };
 
 export async function toArrayBuffer(doc: GLTFTransform.Document):
-    Promise<Uint8Array<ArrayBufferLike>> {
-  return await getIO().writeBinary(doc) as Uint8Array<ArrayBufferLike>;
+    Promise<ArrayBuffer> {
+  return (await getIO().writeBinary(doc)).buffer as ArrayBuffer;
 }
 
-export async function fromArrayBuffer(buffer: Uint8Array<ArrayBufferLike>) {
-  return await getIO().readBinary(buffer);
+export async function fromArrayBuffer(buffer: ArrayBuffer) {
+  return await getIO().readBinary(new Uint8Array(buffer));
 }

--- a/bindings/wasm/lib/import-model.ts
+++ b/bindings/wasm/lib/import-model.ts
@@ -48,8 +48,7 @@ interface Format {
 
 export interface Importer {
   importFormats: Array<Format>;
-  fromArrayBuffer:
-      (buffer: Uint8Array<ArrayBufferLike>) => Promise<GLTFTransform.Document>;
+  fromArrayBuffer: (buffer: ArrayBuffer) => Promise<GLTFTransform.Document>;
 }
 
 /**
@@ -259,7 +258,7 @@ export async function fetchModel(
   const importer = getImporter(options.mimetype ?? uri);
   const response = await fetch(uri);
   const blob = await response.blob();
-  return await importer.fromArrayBuffer(await blob.bytes());
+  return await importer.fromArrayBuffer(await blob.arrayBuffer());
 }
 
 /**
@@ -270,7 +269,7 @@ export async function fetchModel(
 export async function fromBlob(
     blob: Blob, options: ImportOptions = {}): Promise<GLTFTransform.Document> {
   const importer = getImporter(options.mimetype ?? blob.type);
-  return await importer.fromArrayBuffer(await blob.bytes());
+  return await importer.fromArrayBuffer(await blob.arrayBuffer());
 }
 
 /**
@@ -279,8 +278,7 @@ export async function fromBlob(
  * @group Low Level Functions
  **/
 export async function fromArrayBuffer(
-    buffer: Uint8Array<ArrayBufferLike>,
-    identifier: string): Promise<GLTFTransform.Document> {
+    buffer: ArrayBuffer, identifier: string): Promise<GLTFTransform.Document> {
   const importer = getImporter(identifier);
   return await importer.fromArrayBuffer(buffer);
 }
@@ -299,7 +297,7 @@ export async function readFile(filename: string, options: ImportOptions = {}) {
 
   const path =
       filename.startsWith('file:') ? fileURLToPath(filename) : filename;
-  const buffer = await fs.readFile(path);
+  const buffer = (await fs.readFile(path)).buffer as ArrayBuffer;
   return await importer.fromArrayBuffer(buffer);
 }
 

